### ISSUE TITLE
No non-English characters are searched.

### ISSUE THREAD

**Issue by you6878** [drgrib/alfred-bear#9]
<img width="605" alt="Screen Shot 2019-11-26 at 2 21 57 AM" src="https://user-images.githubusercontent.com/23183710/69562931-89b48480-0ff3-11ea-921b-0b89d4990ce5.png">
<img width="605" alt="Screen Shot 2019-11-26 at 2 21 47 AM" src="https://user-images.githubusercontent.com/23183710/69562932-8a4d1b00-0ff3-11ea-92a3-792fc60d422b.png">

Thank you. I'm glad I'm using it.
It doesn't work when I use Korean alphabet.
How can I use non-English characters?

**Comment by AlanRezende** [comment]
When using with latin chars like `ç` or `ã` its does not work too.

**Comment by drgrib** [comment]
The Go code will search for `어`:
```
>> go run cmd/search/main.go "어"
{
    "items": [
        {
            "title": "ㅍㅏﾷ ￜ 어 works",
            "subtitle": "#tech",
            "arg": "B6AA2EED-C378-4759-B7F7-B2618A90AC13-51237-00037885D409A198",
            "valid": true
        }
    ]
}
```

But it appears that Alfred won't send queries that contain it. I suspect Alfred's Script Filter code can only handle ASCII inputs and has an error that blanks out the query once it receives non-ASCII Unicode.

**Comment by bjrnt** [comment]
@drgrib 
I had trouble processing non-ASCII input in my Alfred workflows too, and it turns out it is due to [UTF-8 normalization](https://stackoverflow.com/questions/7931204/what-is-normalized-utf-8-all-about). It is fully possible to search for UTF-8 input as long as you perform NFC normalization on it first.

This is really tricky to notice as the strings will look identical when you print them, but comparisons won't yield the expected result. For example, if I pass "björn" as the Alfred query:
```
utf8.GlyphsInString(os.Args[1]) => 6, because one extra is added due to the ¨ being recorded as a separate glyph
utf8.GlyphsInString("björn") => 5
os.Args[1] == "björn" => false, because the normalization is different
```

Here's quick fix in Go:
```go
import (
    "golang.org/x/text/unicode/norm"
)

query := norm.NFC.String(os.Args[1])
```

**Comment by drgrib** [comment]
Thanks @bjrnt. Fixed in #13.
